src/preset/index.ts

```diff
@@ -65,3 +69,17 @@ export const storyIndexers = async (indexers) => {
     ...(indexers || []),
   ];
 };
+
+/**
+ * Storybook >= 7.4
+ */
+export const experimental_indexers = (indexers) => {
```

Comment:
should this use camelCase to match the others?

Reply:
I agree, but it's the name expected by Storybook: https://github.com/storybookjs/storybook/blob/next/code/lib/core-server/src/utils/getStoryIndexGenerator.ts#L28